code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -197,3 +197,8 @@ GLOBAL_LIST_EMPTY(dummy_mob_list)
 	bound_height = 64
 
 	var/list/extra_bodyparts = list()
+
+/mob/living/carbon/human/dummy/extra_tall/Destroy() //Monkeystation edit
```

Comment:
this should be moved to the modular directory, also the parent call needs to be last